### PR TITLE
chore: use new SupplyAlly branding

### PR DESCRIPTION
**Pull request by sebastianquek**

### CODE REVIEW

File: scripts/commenter.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 PULL_REQUEST_ID=$(echo $CI_PULL_REQUEST | cut -c 59-)
 GITHUB_PROJECT="rationally-app/mobile-application"
-EXPO_PROJECT="@musket/rationally"
+EXPO_PROJECT="@musket/supply-ally"
```
Comment (yehjxraymond) [inline scripts/commenter.sh]: Cannot change this

File: app.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "expo": {
-    "name": "Rationally",
-    "slug": "rationally",
+    "name": "SupplyAlly",
+    "slug": "supply-ally",
```
Comment (yehjxraymond) [inline app.json]: Cannot change this

File: scripts/commenter.sh (hunk 1 above)
Comment (sebastianquek) [inline scripts/commenter.sh]:
```suggestion
EXPO_PROJECT="@musket/rationally"
```

File: app.json (hunk 2 above)
Comment (sebastianquek) [inline app.json]:
```suggestion
    "slug": "rationally",
```

**Pull request merged**